vllm/config.py

```diff
@@ -2753,6 +2759,7 @@ def __repr__(self) -> str:
 @config
 @dataclass
 class LoRAConfig:
+    from vllm.platforms import current_platform
```

Comment:
Let's not do this, importing modules within class body often resolve in side-effect.

For current_platform we can't lazy import, given that we use this extensively to add support for hardware selector accordingly. I'm ok with importing this eagerly.

The proper fix is to fix platforms instead of patching like this.

Reply:
removed. I can make a separate PR to platforms if you can provide any guidance?